### ISSUE TITLE
[Concept] Minor Improvement in Strike/Complaint Record System

### ISSUE THREAD

**Issue by Velmex** [klerikdust/anniediscord#260]
**Idea:** Making it friendly for users, utilizing two prefixes that will record a log of a user. Notably strike being a concrete use for actual punishment for actual rule breaking and complaint being used for notes of possible and/or gray areas of unsatisfactory actions from the user.

--------------------------------------------------------------------------------
```
>strike {id}
+reason {msg}

>complaint {id}
+reason {msg}

```
--------------------------------------------------------------------------------
```
if [strike] then
add "[STRIKE]" + {msg}
else
"[COMPLAINT]" + {msg}
```

--------------------------------------------------------------------------------
Ensure that organized and well-visible "types" within the record log of a user. -also to somehow check how to possibly save and receive the two different types from the same column data. 

_Possible use of strike column from the data without having to make a new column._

**Comment by klerikdust** [comment]
> **Idea:** Making it friendly for users, utilizing two prefixes that will record a log of a user. Notably strike being a concrete use for actual punishment for actual rule breaking and complaint being used for notes of possible and/or gray areas of unsatisfactory actions from the user.
> 
> ```
> >strike {id}
> +reason {msg}
> 
> >complaint {id}
> +reason {msg}
> ```
> 
> ```
> if [strike] then
> add "[STRIKE]" + {msg}
> else
> "[COMPLAINT]" + {msg}
> ```
> 
> Ensure that organized and well-visible "types" within the record log of a user. -also to somehow check how to possibly save and receive the two different types from the same column data.
> 
> _Possible use of strike column from the data without having to make a new column._

@Velmex About the strike & complaint command, they were merged post v6.0.0 patch. In other words, strike now acts just like complaint command but without an automated penalty-action.
Also, speaking of your idea about adding new "+reason" prefix, I thought a simple "+" sign itself would be just obvious enough for user to notice that prefix is used to register a new strike entry.

Nonetheless, I always wanted to improve the user experience whenever possible, so I'll consider to put this in the next updates. Thanks for the suggestion, Vez!

**Comment by Moore2021** [comment]
@klerikdust I talked it over with him, however, we think that possibly make use of the alias. Basically depending on if you use complaint or strike determines how the message is saved. When he says prefix he's not referring to the "+" he's referring to ">strike". so if you wanna record an actual strike it would have strike in bold print and if you just want to make a note and not have it count as a strike you use the alias and "complaint" is added in bold to the reason string.
